macros/src/codegen/module.rs

```diff
@@ -67,29 +54,11 @@ pub fn codegen(
         Context::Idle => {}
 
         Context::HardwareTask(..) => {
-            if let Some(m) = &extra.monotonic {
-                fields.push(quote!(
-                    /// Time at which this handler started executing
-                    pub start: <#m as rtic::Monotonic>::Instant
-                ));
-
-                values.push(quote!(start: instant));
-
-                needs_instant = true;
-            }
+            // None for now.
```

Comment:
Is this a TODO?

Reply:
This is just to copy the `SoftwareTask` match arm when there is nothing to be done there.